<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Lockins</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Reflexion (start it second)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="LockinGui" name="lockin_sig" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Reference (start it first)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="LockinGui" name="lockin_ref" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Fit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Every distance in wavelength unit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Substrate index</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="lineEdit_substrate_index">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_std_substrate_index">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_substrate_index">
                 <property name="text">
                  <string>&lt;no value&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Layer index</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLineEdit" name="lineEdit_layer_index">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_std_layer_index">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_layer_index">
                 <property name="text">
                  <string>&lt;no value&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Angle of incidence</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLineEdit" name="lineEdit_angle">
                 <property name="text">
                  <string>45</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_std_angle">
                 <property name="text">
                  <string>45</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_angle">
                 <property name="text">
                  <string>&lt;no value&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Polarization</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLineEdit" name="lineEdit_polarization">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_std_polarization">
                 <property name="text">
                  <string>360</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_polarization">
                 <property name="text">
                  <string>&lt;no value&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Deposition rate</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Time offset</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLineEdit" name="lineEdit_deposition_rate">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_std_deposition_rate">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_deposition_rate">
                 <property name="text">
                  <string>&lt;no value&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="5" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLineEdit" name="lineEdit_time_offset">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_std_time_offset">
                 <property name="text">
                  <string>120</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_time_offset">
                 <property name="text">
                  <string>&lt;no value&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Global factor</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLineEdit" name="lineEdit_global_factor">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_std_global_factor">
                 <property name="text">
                  <string>100</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_global_factor">
                 <property name="text">
                  <string>&lt;no value&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="XYView" name="graphicsView"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LockinGui</class>
   <extends>QWidget</extends>
   <header>lockin2/src/lockin_gui.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XYView</class>
   <extends>QGraphicsView</extends>
   <header>lockin2/xygraph/xyview.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
